js/repl/Monaco.tsx

```diff
@@ -55,6 +55,7 @@ const asyncPromise = (async function () {
 
 monaco.languages.typescript.typescriptDefaults.setCompilerOptions({
   jsx: monaco.languages.typescript.JsxEmit.React,
+  target: monaco.languages.typescript.ScriptTarget.Latest,
```

Comment:
Set to the latest such that the editor will emit a warning for big int literals.